jsapp/js/account/organization/InviteModal.tsx

```diff
@@ -1,31 +1,103 @@
-import {Group, Modal, Stack, Text, TextInput, ModalProps} from '@mantine/core';
+import {Group, Modal, Stack, Text, TextInput, ModalProps, Loader} from '@mantine/core';
```

Comment:
Linter's asking to break this since it's only importing the type

```suggestion
import type {ModalProps} from '@mantine/core';
import {Group, Modal, Stack, Text, TextInput, Loader} from '@mantine/core';
```